rocAL/rocAL/source/node_image_loader.cpp

```diff
@@ -48,9 +48,13 @@ void ImageLoaderNode::init(unsigned internal_shard_count, const std::string &sou
     reader_cfg.set_batch_count(load_batch_count);
     reader_cfg.set_file_prefix(file_prefix);
     reader_cfg.set_meta_data_reader(meta_data_reader);
+    //  sequence_length, step and stride parameters used only for SequenceReader
+    reader_cfg.set_sequence_length(sequence_length);
```

Comment:
use separate reader_cfg for video and avoid setting video related parameters for image loader

Reply:
We will add this in next PR which will have separate Video Reader class.